test/TritonGPU/amd/amd-conditoinal-barrier.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: triton-opt %s --convert-triton-amdgpu-to-llvm='arch=gfx942' | FileCheck %s
+module attributes {"ttg.compute-capability" = 0 : i32, "ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 8 : i32, "ttg.threads-per-warp" = 64 : i32} {
+  tt.func @conditional_barrier() {
+    // CHECK: llvm.func @conditional_barrier
+    // CHECK: llvm.cond_br
```

Comment:
Can you improve the check here a bit to match with more details with proper indents? Something like the [first few tests here](https://github.com/llvm/llvm-project/blob/main/mlir/test/Dialect/SCF/for-loop-peeling.mlir).